modules/contact-form/tracks-events.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+use Automattic\Jetpack\Tracking;
+
+function tracks_record_grunion_pre_message_sent( $post_id, $all_values, $extra_values ) {
+	$tracking = new Automattic\Jetpack\Tracking();
+	if ( $extra_values['is_block'] ) {
+		$tracking->tracks_record_event( wp_get_current_user(), 'jetpack_contact_form_gutenberg_block_message_sent', array(
```

Comment:
Do we need `gutenberg` in the event name here? Wouldn't `block` mean that this comes from a form block?

Reply:
No, good call. I have removed it.